go/cli/mcap/cmd/filter.go

```diff
@@ -64,6 +64,7 @@ func buildFilterOptions(flags filterFlags) (*filterOpts, error) {
 	case "lz4":
 		opts.compressionFormat = mcap.CompressionLZ4
 	case "none":
+	case "":
```

Comment:
why this change? i figure it's clearer to require a non-empty string if `--compression` is supplied.

Reply:
I tried to run the command `mcap compress --compression "" input.mcap -o output.mcap` and it failed, telling me to use `"none"` instead. Not a terrible outcome (it told me what to do), but why not just do what I asked and use the `""` compression mode as defined in the MCAP spec?